### DISCUSSION TITLE
With VmWare Worstation 17 PRO resolution

### DISCUSSION THREAD

**Discussion by jfauss** [elementary/triage#198] [Q&A] [answered]
Elementary OS well installed. Having a little screen 13x9cm. Is there a possibility to install vmware tools to have a big screen like the size of monitor ?

**Comment by lenemter** [comment]
When I installed elementary OS in Virtualbox I was able to change the resolution in the display settings. Can you do it or you can't select a required resolution?

**Reply by jfauss** [reply]
It's OK. I find the solution : installing as ubuntu-64 in vmware and installaing the vmware tools manually.
It's not a change of resolution in display settings. it's the ability of vmware to display correctly with its own tools named "vmware tools" : linux packages.

**Comment by jfauss** [comment, marked as answer]
see previous answer
